Purchases/RCBackend.h

```diff
@@ -32,7 +31,7 @@ typedef void(^RCBackendPurchaserInfoResponseHandler)(RCPurchaserInfo * _Nullable
 typedef void(^RCIntroEligibilityResponseHandler)(NSDictionary<NSString *,
                                                  RCIntroEligibility *> *);
 
-typedef void(^RCEntitlementResponseHandler)(RCEntitlements * _Nullable, NSError * _Nullable);
+typedef void(^RCOfferingsResponseHandler)(NSDictionary * _Nullable, NSError * _Nullable);
```

Comment:
This is `NSDictionary` and not `RCOfferings`?

Reply:
Yes, because the initialization happens in RCPurchases.m, which is where the products are retrieved. If don't have the products in the RCBackend so I can't build the RCOfferings.